test/cypress/features/transaction.feature

```diff
@@ -0,0 +1,39 @@
+Feature: Transactions 
+
+  Background:
+    Given I login as testnet_guy on testnet
+    And I wait 5 seconds
+    Given I am on Transactions page
+
+  Scenario: Load Latest Transactions
+    Then I should see 20 transactions in table
+    When I click on showMoreButton
+    Then I should see more than 20 transactions in table
+
+  Scenario: Filter by one amount then add second and third filters for another amount and sender respectively
+    When I click on filterTransactionsBtn
+    Then I should see filterDropdown
+    When I fill 100 in amountFromInputFilter field
+    And I click on applyFilters
+    Then I should see 20 transactions in table
+    When I click on filterTransactionsBtn
+    Then I should see filterDropdown
+    When I fill 1000 in amountToInputFilter field
+    And I click on applyFilters
+    Then I should see 20 transactions in table
+    When I click on filterTransactionsBtn
+    Then I should see filterDropdown
+    When I fill lskgpfj26e3uogvr5wbono75hzn5myds4pncv3zm8 in senderAddressFilter field
+    Then I should see more than 5 transactions in table
```

Comment:
I think a deterministic number of transactions can also be known here if the test is ran against devnet

Reply:
@eniolam1000752 I've explained my thought process behind this in the step definition. I'm not quite certain that running against devnet might affect the process too much but I'll look into it